src/stacks-hierarchy/Root_TabNavigatorStack/TabNav_DashboardStack/Dashboard_TripSearchScreen/components/CityZoneMessage.tsx

```diff
@@ -49,11 +55,13 @@ export const CityZoneMessage: React.FC<CityZoneMessageProps> = ({
     }
   };
 
-  const messageActions = selectedCityZones.map((cityZone) => ({
-    id: `${cityZone.id}_action`,
-    text: cityZone.name,
-    onPress: () => openUrlForCityZone(cityZone),
-  }));
+  const messageActions = [
+    {
+      id: `${fromCityZone.id}_action`,
+      text: fromCityZone.name,
+      onPress: () => openUrlForCityZone(fromCityZone),
+    },
+  ];
```

Comment:
Will there ever be more than one messageAction now? If so, maybe we could simplify the logic here and in `CityZoneBox`.

Reply:
Not now, but I am working another ticket that requires more than one message action